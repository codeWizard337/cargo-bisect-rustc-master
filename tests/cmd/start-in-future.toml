bin.name = "cargo-bisect-rustc"
args = "--start 9999-01-01"
status = "failed"
