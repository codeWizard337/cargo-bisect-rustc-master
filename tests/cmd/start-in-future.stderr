ERROR: start date should be on or before current date, got start date request: 9999-01-01 and current date is [..]
